### DISCUSSION TITLE
Since Kong can download docker images of arm，when can apisix in docker-hub can support linux/arm64/v8 ?

### DISCUSSION THREAD

**Discussion by Myicefrog** [apache/apisix#4861] [General]
More and more companies choose machines based on arm and they would like to use docker images directly，so  it is better to push images to docker hub including amd64 and arm64

**Comment by spacewander** [comment]
Here is it: https://hub.docker.com/layers/apache/apisix/2.8-alpine/images/sha256-ae303eb5c3b77e87911a11a899bd2f3e3e538ccabbc650fd840cd3673380f56c?context=explore

**Reply by renshunhang** [reply]
> Here is it: https://hub.docker.com/layers/apache/apisix/2.8-alpine/images/sha256-ae303eb5c3b77e87911a11a899bd2f3e3e538ccabbc650fd840cd3673380f56c?context=explore

Hi, could you provide me with a link to the ARM image of APISIX version 3.11.0? Thank you.

**Reply by SkyeYoung** [reply]
@renshunhang u can check the tags here https://hub.docker.com/r/apache/apisix/tags
